scripts/deploy.sh

```diff
@@ -70,6 +70,8 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
     sed -i "s/maas_notification_plan: .*/maas_notification_plan: npTechnicalContactsEmail/" $RPCD_VARS
     # set network speed for vms
     echo "net_max_speed: 1000" >>$RPCD_VARS
+    # Enable host security hardening for AIO builds
+    sed -i "s/apply_security_hardening: .*/apply_security_hardening: true/" $RPCD_VARS
```

Comment:
We need to be able to choose whether or not this is enabled. AIO builds are used by gating and so this would interfere with testing.